docs/csharp/language-reference/statements/exception-handling-statements.md

```diff
@@ -95,6 +95,43 @@ You can provide several `catch` clauses for the same exception type if they dist
 
 If a `catch` clause has an exception filter, it can specify the exception type that is the same as or less derived than an exception type of a `catch` clause that appears after it. For example, if an exception filter is present, a `catch (Exception e)` clause doesn't need to be the last clause.
 
+##### Exception filters vs. traditional exception handling
+
+Exception filters provide significant advantages over traditional exception handling approaches. The key difference is **when** the exception handling logic is evaluated:
+
+- **Exception filters (`when`)**: The filter expression is evaluated *before* the stack is unwound. This means the original call stack and all local variables remain intact during filter evaluation.
+- **Traditional `catch` blocks**: The stack is unwound *before* entering the catch block, potentially losing valuable debugging information.
```

Comment:
"Before" is italicized in both bullet points to add emphasis, which doesn't make much sense since they're trying to point out differences.